parser/preprocessed/SqlLexer_gen.cpp

```diff
@@ -8,8 +8,8 @@
 
 #define FLEX_SCANNER
 #define YY_FLEX_MAJOR_VERSION 2
-#define YY_FLEX_MINOR_VERSION 6
-#define YY_FLEX_SUBMINOR_VERSION 0
+#define YY_FLEX_MINOR_VERSION 5
```

Comment:
Good point @rogersjeffreyl on noting it in the developer workflow. Done! Changes are reflected in https://cwiki.apache.org/confluence/display/QUICKSTEP/Workflow+For+Committer